test/test.mjs

```diff
@@ -24,7 +24,7 @@ test('equal fail', t => t.equal('a', 'b'))
     assert.strict.equal(didPass, false)
     assert.strict.equal(title, 'equal fail')
     assert.strict.equal(location, 'test.mjs:22')
-    assert.strict.equal(message.split('\n').length, 2)
+    assert.strict.equal(typeof message, 'string')
```

Comment:
This was necessary because assertions errors contain more newlines in Node 12 than they do in other versions of Node. I made the assertions less specific, assuming that an assertion regarding the contents of assertion errors generated by Node isn't all that important.

Reply:
Good call making these less picky about the error text. 👍